Gsk-4.0.gir

```diff
@@ -2003,7 +2003,7 @@ during deserialization of node data.</doc>
       <parameters>
         <parameter name="section" transfer-ownership="none">
           <doc xml:space="preserve">the #GtkCssSection where the error occurred</doc>
-          <type c:type="gconstpointer" name="gconstpointer"/>
```

Comment:
```
GirXml: ./Gsk-4.0.gir at line 2006:11: <type> element is missing a name attribute
```

Reply:
We should really just ignore these types in gir, and comment out any API related to it.

Reply:
Indeed, for now I have added it to the missing/broken annotations issue. It's getting a bit bigger, I will have to get around those and fix them as soon as possible as well.

Reply:
For now, I have added it to fix.sh, hopefully it's fine now.

Reply:
So it seems :)

Reply:
I will try to regenerate the bindings and see how that goes as well.

Reply:
Let me know if you want me to take a look over the changes

Reply:
It seems we will need to add `<include name="Gtk" version="4.0"/>` to gsk4 for `Gtk.CssSection` to be picked properly.  See this PR for the changes https://github.com/gtk-rs/gtk4-rs/pull/41

Reply:
That's problematic. Then we have gtk4 depending on gsk4 and in reverse :) Does that even compile?

Reply:
Agh yes, that doesn't even compile. I guess we will just ignore it for now?

Reply:
Yes and make sure to report that to GTK ASAP so this is fixed before 4.0.0

Reply:
Removed the ParseErrorFunc as I couldn't ignore it from the -sys crate & reported both issues 
https://gitlab.gnome.org/GNOME/gtk/-/issues/3326 & https://github.com/gtk-rs/gir/issues/982

Reply:
Please don't add this to `fix.sh` and wait for gir fix instead.

Reply:
I don't mind having it in fix.sh until the gir fix is done. It would be good to move forward here with GTK4 so we can find further issues before 4.0.0 is released and can make sure they get fixed, blocking on a perfect solution is only going to delay that and then the API might be frozen already.

Reply:
I don't see why they shouldn't be added to fix.sh? The issue has been upstream for a long time (the one i opened seems to be a duplicate)

Reply:
Because adding a fix in `fix.sh` isn't necessary here if I understood correctly. The "good" fix would be to add the item to the gir ignore list (which is currently broken).

Reply:
what about just renaming the GtkCssSection to a gpointer, would that fit as part of the fix.sh? it used to be there iirc, it just needs to be updated

Reply:
Works for me

Reply:
Done, I've re-generated the bindings as well https://github.com/gtk-rs/gtk4-rs/pull/41